components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__phenotype.xml

```diff
@@ -169,13 +169,13 @@
   }
 
   $$('.chapter.phenotype-info .collapsible-group h3').each(function(item){
-     if (item.up('.browser') &amp;&amp; !item.up('.collapsible-group').down('input[type=checkbox]:checked')) {
-       toggleCategory(item);
-     }
-     item.observe('click', function() {
-       toggleCategory(item);
-       relayoutSummaryStickyBox();
-     });
+    if (item.up('.browser') &amp;&amp; !item.up('.collapsible-group').down('input[type=checkbox]:checked')) {
+      toggleCategory(item);
+    }
+    item.observe('click', function() {
+      toggleCategory(item);
+      relayoutSummaryStickyBox();
+    });
```

Comment:
These are formatting changes, not related to fixing this issue, and they should be in a separate PR.